libs/features/cdt/cdt.js

```diff
@@ -0,0 +1,118 @@
+import { getMetadata, getConfig, createTag } from '../../utils/utils.js';
+import { replaceKey } from '../placeholders.js';
+
+function loadCountdownTimer(
+  container,
+  cdtLabel,
+  cdtDays,
+  cdtHours,
+  cdtMins,
+  timeRangesEpoch,
+) {
+  let isVisible = false;
+  let interval;
+
+  function appendTimerBox(parent, value, label) {
+    const fragment = createTag('div', { class: 'timer-fragment' }, null, { parent });
+    const unitContainer = createTag('div', { class: 'timer-unit-container' }, null, { parent: fragment });
+    createTag('div', { class: 'timer-unit-label' }, label, { parent: fragment });
+
+    createTag('div', { class: 'timer-box' }, Math.floor(value / 10).toString(), { parent: unitContainer });
+    createTag('div', { class: 'timer-box' }, (value % 10).toString(), { parent: unitContainer });
+  }
+
+  function appendSeparator(parent) {
+    createTag('div', { class: 'timer-separator' }, ':', { parent });
+  }
+
+  function appendTimerBlock(parent, daysLeft, hoursLeft, minutesLeft) {
+    const timerBlock = createTag('div', { class: 'timer-block' }, null, { parent });
+    appendTimerBox(timerBlock, daysLeft, cdtDays);
+    appendSeparator(timerBlock);
+    appendTimerBox(timerBlock, hoursLeft, cdtHours);
+    appendSeparator(timerBlock);
+    appendTimerBox(timerBlock, minutesLeft, cdtMins);
+  }
+
+  function appendTimerLabel(parent, label) {
+    createTag('div', { class: 'timer-label' }, label, { parent });
+  }
+
+  function removeCountdown() {
+    container.innerHTML = '';
+  }
+
+  function render(daysLeft, hoursLeft, minutesLeft) {
+    if (!isVisible) return;
+
+    removeCountdown();
+
+    appendTimerLabel(container, cdtLabel);
+    appendTimerBlock(container, daysLeft, hoursLeft, minutesLeft);
+  }
+
+  function updateCountdown() {
+    const currentTime = Date.now();
```

Comment:
```suggestion
    const instant = new URL(window.location.href)?.searchParams?.get('instant');
    const currentTime =  instant ? new Date(instant) : Date.now();
```

Reply:
@rahulgupta999 i think @afmicka is right and this will be required, so we should probably implement it now, see above suggestion that should do it

Reply:
Done.